sim/aviron/src/lib/io.zig

```diff
@@ -191,3 +161,32 @@ pub const IO = struct {
         return null;
     }
 };
+
+// Unified byte-addressable device interface used by MemorySpace for RAM and IO
+pub const Device = struct {
```

Comment:
I would rename `Device` to `Bus` as it's not a "device" at all, but an abstraction over a memory bus

Reply:
Done